Doc/whatsnew/3.8.rst

```diff
@@ -614,6 +614,10 @@ Changes in the Python API
   specialized methods like :meth:`~tkinter.ttk.Treeview.selection_set` for
   changing the selection.  (Contributed by Serhiy Storchaka in :issue:`31508`.)
 
+* The :meth:`writexml`, :meth:`toxml` and :meth:`toprettyxml` methods
+  now preserves the attribute order specified by the user.
```

Comment:
please list which modules have been modified. Something like: "xml.dom.minidom, xml.etree, ...: The ... methods ...".